
# HG changeset patch
# User Whoopie
# Date 1407938863 -28800
# Node ID 939944f0fca9b2dcdf8470bb24efcc37a3843e8b
# Parent  f7f6c15b0ec30b0def69c9ccbb3dc4eb97374abc
Fix wtmp, testing for wtmp.h and wtmpx.h doesn't make sense

diff -r f7f6c15b0ec3 -r 939944f0fca9 loginrec.h
--- loginrec.h
+++ loginrec.h
@@ -79,10 +79,10 @@
 #  if defined(HAVE_UTMP_H) && defined(UTMP_FILE) && !defined(DISABLE_UTMP)
 #    define USE_UTMP
 #  endif
-#  if defined(HAVE_WTMPX_H) && defined(WTMPX_FILE) && !defined(DISABLE_WTMPX)
+#  if defined(WTMPX_FILE) && !defined(DISABLE_WTMPX)
 #    define USE_WTMPX
 #  endif
-#  if defined(HAVE_WTMP_H) && defined(WTMP_FILE) && !defined(DISABLE_WTMP)
+#  if defined(WTMP_FILE) && !defined(DISABLE_WTMP)
 #    define USE_WTMP
 #  endif
 
